intelmq/bots/collectors/xmpp/collector.py

```diff
@@ -1 +1,52 @@
-''' TBD '''
\ No newline at end of file
+import os.path
+import time
+import json
+import redis
+from sleekxmpp import ClientXMPP
+from sleekxmpp.exceptions import IqError, IqTimeout
+from intelmq.lib.bot import Bot, sys
+
+class XMPPCollectorBot(Bot): 
+
+    xmpp=None
+    
+    def process(self):  
+             
+        if self.xmpp==None:	
+	 self.xmpp = ACDCXMPPBot(self.parameters.xmpp_user+'@'+self.parameters.xmpp_server, self.parameters.xmpp_key,self.send_message,self.logger)
+         self.xmpp.connect(reattempt=True)
+         self.xmpp.process()
+      
+	
+
+    def killbot(self):
+	self.xmpp.disconnect(wait=True)	
+	self.logger.info("Disconnected") 	
+	
+	
+
+class ACDCXMPPBot(ClientXMPP):
+        
+        def __init__(self,jid,password,objects,objectl):
```

Comment:
`objects` and `objectl` are not good variable names. Why are these parameters at all? Shouldn't they be `send_message` and `logger` all the time?